### ISSUE TITLE
Multiple books => Duplicate learning objective content in the Summary section

### ISSUE THREAD

**Issue by kerwinso** [openstax/cnx-recipes#214]
1. Go to https://dev00.cnx.org/contents/GFy_h8cu@10.61:2zzm1QG9@7/34-Proteins.
1. Click the Summary link with the triangle near the top of the page.

Note that the content is a duplicate of the learning objectives in the main body of the page.

![image](https://cloud.githubusercontent.com/assets/11298165/26366192/bebe954c-3faf-11e7-811d-f8ec98b3df5b.png)

**Comment by helenemccarron** [comment]
The ruleset eliminates the abstract/summary but still appears in webview.

**Comment by kerwinso** [comment]
Also an issue in University Physics.

**Comment by openstaxalina** [comment]
We have toyed with the idea of moving the summary **out** of the metadata (< abstract >) and **into** the body of the content, but it would require a content change.  I think it would be best to leave the summary in the metadata/abstract and **not** duplicate it in the content body.  But I may not know the full history - we can discuss further if needed.

**Comment by reedstrm** [comment]
Currently, the recipe does the copying of  the summary into the body. Note that we have a conceptual issue with the metadata - nothing inside the <metadata> tag is interpreted (except for composite-pages/chapters) when the baked HTML is re-interpreted as pages and stored in the database. This means that the only metadata that can be overridden (_i.e._ differ in baked vs. raw) is the title of a page. This is controlled by the title that is _outside_ the metadata tags. It would be wrong to let the baking process change the metadata of the published page.

**Comment by openstaxalina** [comment]
_**The Plan:**_

1) **Tech** For OpenStax books cooked with recipes, hide the Summary drop-down because the 
< abstract > field is either empty or being duplicated within the content tag.

2) **Content** Later on, move the learning objectives from the < abstract > and into the content tag.

3) **Tech** Later-later on, re-enable the Summary drop-down in case we can ever add anything actually useful there.

**Comment by reedstrm** [comment]
Further discussion with @helenemccarron and @tomjw64 lead to leaving this in place, but finding a way to suppress display of the summary _per book_.  We'll need a way to pass a whole-document level style option, store it, and return it. with the json. Will discuss design choices with @philschatz

**Comment by kerwinso** [comment]
The summary toggle for baked books has been hidden with this PR: https://github.com/Connexions/webview/issues/1574. 
Now only 1 set of learning objectives is displaying, which is the desired behavior:

![image](https://user-images.githubusercontent.com/11298165/28332409-4a9bda6c-6bba-11e7-9e0f-31c3303fec85.png)

I will leave this issue open so Ross et al can investigate the long-term fix.

**Comment by kerwinso** [comment]
This was temporarily fixed for the initial rollout milestone by hiding the summary toggle. However, a longer term fix is still required, which is why this issue is still open.

**Comment by openstaxalina** [comment]
We can regression test with the summary hidden.

We need to move the summaries from the < abstract > to the  < content  > when we have batch publish.  

Then we will no longer need to suppress the summary toggle.

**Comment by openstaxalina** [comment]
Summaries were moved from the < abstract > to the < content > and are now displaying only once (correctly): 
https://cnx.org/contents/jVCgr5SL@15.3:zmz95BrD@12/1-1-The-Science-of-Biology
https://cnx.org/contents/eg-XcBxE@15.2:g5baFV-k@5/1-1-Temperature-and-Thermal-Equilibrium

Closing.

(👋 Kerwin if you're seeing this!)
